locale/en_US/manager.po

```diff
@@ -2514,10 +2514,13 @@ msgstr "Monthly"
 msgid "stats.pdf"
 msgstr "PDF"
 
-msgid "stats.publications.downloadReport.betweenDates"
+msgid "stats.geographic.ccAttribution"
+msgstr "<a href='https://db-ip.com'>IP Geolocation by DB-IP</a>"
```

Comment:
@asmecher Bozana said that she spoke with you about the CC attribution for IP geolocation. Do we need to put it in such a prominent place in the UI?

![geolocation-attribution](https://user-images.githubusercontent.com/2306629/197550376-e1cf1d23-b1fa-4eb1-806d-042d741848c9.png)

Would it be acceptable to put it in with the stats settings instead, like this?

![geolocation-attribution-in-settings](https://user-images.githubusercontent.com/2306629/197550937-c202e4da-2513-4246-a367-a653186f2850.png)

Reply:
The license for it says:
> You are free to use this IP to City Lite database in your application, provided you give attribution to DB-IP.com for the data.
>
> In the case of a web application, you must include a link back to DB-IP.com on pages that display or use results from the database. You may do it by pasting the HTML code snippet below into your code

Thus wherever we present results that draw from that library we have to also present a link to the source.

Reply:
Hmm, I guess that means the CSV file itself, then. Right? Because we don't show the geographic stats anywhere except in the CSV file.

Reply:
Yes, till now it is only the API (CSV or JSON) and that Statistics > Articles/Monographs/Preprints page, where the user can download the CSV.
Regarding the API I would agree with Alec: "...APIs are meant to be machine-readable and the license appears to be talking about human-facing content".
I would think it is sufficient to give the attribution near the Download-Button in the UI (and e.g. in the API documentation), but I am not sure if we would need to put it somehow into the downloaded CSV-file (which somehow feels a little bit strange)?

Reply:
Ok, I guess we can't get around it. It just sticks out like a sore thumb there. Here are two alternatives.

A tooltip that says "Geolocation provided by [DB-IP](#)."

![tooltip](https://user-images.githubusercontent.com/2306629/197750750-6608cfc1-eb6f-45bc-9a3a-cc6a9873dc7c.png)

Or as a small footer at the bottom of the article stats page.

![footer](https://user-images.githubusercontent.com/2306629/197750785-4b698623-59ae-4f11-8522-41b4f44c5490.png)

Reply:
If no objection I would choose the tooltip for now...

Reply:
@NateWr, what should be the tooltip label?

Reply:
^ I'm OK with that solution as well. I think we can meet the terms without such a glaring diversion from our usual "neutral" UI practices.

Reply:
@bozana The tooltip should read:

> Geolocation provided by [DB-IP](#).

Reply:
Also the label part, for the screenreader?

Reply:
It can be something like "About Geographic"